tests/test_authenticator.py

```diff
@@ -2166,6 +2167,8 @@ def test_authentication_flow_document(self):
         provider.identifier_maximum_length=22
         provider.password_maximum_length=7
         provider.identifier_barcode_format = provider.BARCODE_FORMAT_CODABAR
+        FLOW_TYPE_BASIC = 'http://opds-spec.org/auth/basic'
```

Comment:
I also define this in First Book tests. Is this fine as it is or should I move these constants to `BasicAuthenticationProvider`?